src/page/AddMenu/components/MenuName/index.tsx

```diff
@@ -1,29 +1,50 @@
 import useMediaQuery from 'utils/hooks/useMediaQuery';
+import useAddMenuStore from 'store/addMenu';
 import styles from './MenuName.module.scss';
 
-export default function MenuName() {
+interface MenuNameProps {
+  isComplete: boolean;
+}
+
+export default function MenuName({ isComplete }: MenuNameProps) {
   const { isMobile } = useMediaQuery();
+  const { name, setName } = useAddMenuStore();
+  const handleNameChange = (e : React.ChangeEvent<HTMLInputElement>) => {
+    setName(e.target.value);
+  };
   return (
     <div>
       {isMobile ? (
         <div className={styles.mobile__container}>
           <div className={styles.mobile__caption}>
             메뉴명
           </div>
-          <input
-            className={styles.mobile__nameInput}
-            placeholder="예) 불족발 + 막국수 저녁 SET"
-          />
+          {isComplete ? (
+            <span className={styles['mobile__name-text']}>{name}</span>
+          ) : (
+            <input
+              className={styles.mobile__nameInput}
```

Comment:
띄어쓰기 네이밍 컨벤션 지켜주세요! (아래도 변경이 필요해보이네요)
```suggestion
              className={styles.mobile__name-Input}
```

Reply:
수정 완료 했습니다.
77dad0277c5be201590bb02f7ca573596958ff69